dll/win32/shell32/CDefView.cpp

```diff
@@ -1884,6 +2271,13 @@ LRESULT CDefView::OnCommand(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHand
 
     TRACE("(%p)->(0x%08x 0x%08x %p) stub\n", this, dwCmdID, dwCmd, hwndCmd);
 
+    if (dwCmdID >= DVIDM_ARRANGESORT_FIRST && dwCmdID <= DVIDM_ARRANGESORT_LAST)
+    {
+        UINT listCol = (UINT) GetMenuItemDataById(m_hMenuArrangeModes, dwCmdID);
```

Comment:
```suggestion
        UINT listCol = (UINT)GetMenuItemDataById(m_hMenuArrangeModes, dwCmdID);
```

Reply:
Where in the coding style document does it say this is required?

Reply:
It is not enforced in the coding-styles, but it usually saves line-width. It's a suggestion, not more.